packages/flex-plugin/src/utils/loadCSS.ts

```diff
@@ -0,0 +1,12 @@
+export const loadCSS = (href: string) => {
```

Comment:
Methods should accept an array `loadCSS = (...href: string[])`, since developers can have:

```ks
loadCSS('/my-single-css.css');
// or
loadCSS(
   'file1.css',
   'file2.css',
);
```